src/views/Log.vue

```diff
@@ -142,46 +142,44 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     </v-container>
 
     <!-- the log file viewer -->
-    <v-row
+    <div
+      ref="logScroll"
```

Comment:
I had to change this to a native HTML element in order to be able to use https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollBy